### PR TITLE
Entity class specified

### PR DESCRIPTION
**Pull request by hokka1do**
Entity class created and added to project

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,6 +26,18 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <version>2.2.6.RELEASE</version>
```
Comment (iChurinov) [inline pom.xml]: Why do you need exactly this version?

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.hokka1do.bookme.professionals;
+
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.persistence.*;
+
+@Entity
+@Table
```
Comment (iChurinov) [inline src/main/java/com/hokka1do/bookme/professionals/Professional.java:8]: What is the purpose of this annotation here?

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    @GetMapping("/getLastName")
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    private String lastName;
```
Comment (iChurinov) [inline src/main/java/com/hokka1do/bookme/professionals/Professional.java]: Class fields must be declared before methods
https://stackoverflow.com/a/12427127

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    @GetMapping("/getLastName")
```
Comment (iChurinov) [inline src/main/java/com/hokka1do/bookme/professionals/Professional.java]: Why is that here? Does it work for HTTP calls?

**Review by iChurinov — COMMENTED** [review]
Do not see how to add or retrieve `Professional` resource info in your application

**Comment by iChurinov** [comment]
Branch names should be a short meaningful description of the code change.   
Rename branch `new_feature1` to something like `adding-and-retrieving-professional`

File: pom.xml (hunk 1 above)
Comment (hokka1do) [reply pom.xml]: Deleted

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java (hunk 2 above)
Comment (hokka1do) [reply src/main/java/com/hokka1do/bookme/professionals/Professional.java:8]: Purpose of this annotation is to tell spring that this column is a column in database.

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java (hunk 3 above)
Comment (hokka1do) [reply src/main/java/com/hokka1do/bookme/professionals/Professional.java]: Fixed .Accidentally

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
+package com.hokka1do.bookme.professionals;
+
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.persistence.*;
```
Comment (iChurinov) [inline src/main/java/com/hokka1do/bookme/professionals/Professional.java:5]: Wildcard imports should be avoided
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java (hunk 4 above)
Comment (hokka1do) [reply src/main/java/com/hokka1do/bookme/professionals/Professional.java]: Nope erased.

File: src/main/java/com/hokka1do/bookme/professionals/Professional.java (hunk 5 above)
Comment (hokka1do) [reply src/main/java/com/hokka1do/bookme/professionals/Professional.java:5]: Got it. Will check in a future.
